### ISSUE TITLE
Demo for Swipe_RecyclerView

### ISSUE THREAD

**Issue by u2giles** [rahulrj/Swipe_RecyclerView#1]
Is there a sample for  Swipe_RecyclerView  similar to https://github.com/47deg/android-swipelistview-sample ?

**Comment by rahulrj** [comment]
Yes, its already a part of this project. I have modified the above mentioned sample to work with a RecyclerView

**Comment by u2giles** [comment]
Thanks I will try it but first I am a bit confuse how to import this library in Eclipse. I am not using Studio so I have to import your library as a "project library". When I import the main folder, I have the following choices : "SwipeListViewExampleActivity", "debug", "release", "TestActivity". "21.0,0" and "main". Could you please tell me which project contains your library. Or do you suggest another procedure to use your library in EclipseADT? Thanks in advance. Gilles

**Comment by rahulrj** [comment]
I have removed the build folder in the library and now you should be able to import the library 'swipelistview' separately in Eclipse. I tried and its imported successfully. And for importing the project, just remove the 'swipelistview' folder from it and while importing select only 'import src'.

**Comment by u2giles** [comment]
Hi  Rahulrj. I eventually made your demo works in Eclipse after struggling for a while.
In fact, I had to rebuilt both the lib project and the demo project manually from scratch . I also had to add nineoldandroids-2.4.0 as  I could not find it in your zip install. I hope it's a compatible version but so far so good.
I definitely need to move to gradle/studio as soon as possible to avoid these kind of problems in the future.
Thanks Again for the update of your library. Great Library !
